cpmonitor/migrations/0029_move_chart_images_and_resize_images.py

```diff
@@ -0,0 +1,68 @@
+# Generated by Django 4.1.7 on 2023-09-09 13:17
+
+from django.db import migrations, models
+from pathlib import Path
+import os
+import shutil
+
+from PIL import Image
+
+
+def resize_images(apps, schema_editor):
+    upload_path = Path("cpmonitor") / "images" / "uploads"
+
+    if not os.path.exists(upload_path):
+        return
+
+    if not os.path.exists(upload_path / "charts"):
+        os.mkdir(upload_path / "charts")
+
+    for root, dirs, files in os.walk(upload_path):
```

Comment:
I would prefer `basedir, _, files`. `root` is somewhat misleading, and `_` shows, it is unused.

Reply:
I used the default example from the python docs: https://docs.python.org/3/library/os.html. They call it root. If you prefer basedir I don't mind.